recipes/open-color

```diff
@@ -0,0 +1 @@
+(open-color :repo "a13/open-color.el" :fetcher github :files "open-color.el")
```

Comment:
You have a couple of options here.  This is what you intended, but is corrected (`:files` must be a list):
```elisp
(open-color :repo "a13/open-color.el" :fetcher github :files ("open-color.el"))
```
but ultimately this is equivalent to the default recipe, which MELPA prefers:
```suggestion
(open-color :repo "a13/open-color.el" :fetcher github)
```